garnett/utils.py

```diff
@@ -16,33 +19,47 @@ def get_default_language():
     else:
         default = setting
 
-    if isinstance(default, str):
+    if isinstance(default, Language):
         return default
+    elif isinstance(default, str):
+        return Language.get(default)
     else:
         raise ImproperlyConfigured(
-            "GARNETT_DEFAULT_TRANSLATABLE_LANGUAGE must be a string or callable that returns a string"
+            "GARNETT_DEFAULT_TRANSLATABLE_LANGUAGE must be a string or callable that returns a string or `Language` object"
```

Comment:
Thoughts on logging this instead and failing a little less loudly. Probably don't make sense in this PR, just wanted to open discussion.